[static-runtime] (wip) modify variadic op processor to work with nodes which produce multiple outputs

tenpercent opened the pull request:
Differential Revision: D35582534

mikeiovine reviewed (approved):
Reviewed internally

tenpercent committed:
[static-runtime] modify variadic op processor to work with nodes which produce multiple outputs (#75672)

Summary: Pull Request resolved: https://github.com/pytorch/pytorch/pull/75672

Test Plan: Unit tests

Reviewed By: mikeiovine

Differential Revision: D35582534

fbshipit-source-id: e4203f132218b5c8e4d3db8e180d4306d1a2288c
